signer/Cargo.toml

```diff
@@ -15,6 +15,7 @@ bench = false
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
```

Comment:
I cannot comment on code you didn't write (probably for the best) but I think we should call this package `signer` and not `sbtc-signer`. To run cargo tests on this package we need to run this:

```bash
cargo test --package sbtc-signer
```

Where every external indicator makes it look like it should be `cargo test --package signer`.

Reply:
Yeah, that makes sense to me. I think I just neglected to change this when we settled on calling this crate `signer`.

Reply:
Yeah I 100% agree. Do we have a ticket to change this? I'm happy to do it, but I prefer to do it as a separate change to keep our PRs focused and commit history clean.

Reply:
Issue created, I'll have it done as soon as we've merged this PR. https://github.com/stacks-network/sbtc/issues/142